src/main/java/Duke.java

```diff
@@ -1,10 +1,45 @@
+import java.util.Scanner;
+
+import duke.duke_exception.DukeException;
+import duke.tasklist.TaskList;
+import duke.utility.parser.CommandMap;
+import duke.utility.parser.Parser;
+import duke.utility.storage.Storage;
+import duke.utility.ui.Ui;
+import duke.utility.ui.UiMessage;
+
+/**
+ * <h1>Duke Chatbot</h1> The Duke chatbot is a bot that is capable to keep track of tasks from the
+ * users.
+ * 
+ * @author Brian Quek
+ */
+
 public class Duke {
+
+    /**
+     * Main function that creates and run the Duke Bot object.
+     * 
+     * @param args
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner inputScanner = new Scanner(System.in);
```

Comment:
As the UI class serves to deal with interactions with the user, maybe the scanner that reads user inputs could be abstracted into the UI class?

Reply:
1. I feel that UI's main job should be dealing with what to printout and not the interaction as itself. I didn't want to put too many things into the UI class itself but I will take note and adjust accordingly at the very end of the iP.

2. Yeap the 5-8 are removed line so they have been moved into the UI class.